src/Datadog.Trace/Util/DomainMetadata.cs

```diff
@@ -8,6 +8,7 @@ namespace Datadog.Trace.Util
     /// </summary>
     internal static class DomainMetadata
     {
+        private const string IsAppInsightKey = "DD_IsAppInsight";
```

Comment:
Do we already have a convention for this? If not, would we consider  "DataDog.IsAppInsights".
And then use "DataDog." prefix for all this settings, environment variables etc..?
SUch settings are, essentially, public APIs because they may conflict with customer data.
Regardless of that , AppInsights has an s at the end :)

Reply:
Oh, sorry, just learned that DD_ prefix _is_ the standard convention. Never mind :)